apps/earn-protocol/app/portfolio/[walletAddress]/page.tsx

```
@@ -69,16 +122,31 @@
     (activity) => activity.account.toLowerCase() === walletAddress.toLowerCase(),
   )
 
+  const positionsHistoricalChartMap = positionsWithVault.reduce<{
+    [key: string]: HistoryChartData
+  }>(
+    (acc, position) => ({
+      ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)